client/shipping-label/state/actions.js

```diff
@@ -70,18 +99,29 @@ export const purchaseLabel = () => ( dispatch, getState, { callbackURL, nonce, s
 		}
 	};
 
-	const form = getState().shippingLabel.form;
-	const formData = {
-		origin: form.origin.values,
-		destination: form.destination.values,
-		payment_method_id: '123456789',
-		carrier: 'usps',
-		packages: form.packages.values.map( ( pckg, index ) => ( {
-			...omit( pckg, [ 'items', 'id' ] ),
-			service_id: form.rates.values[ index ],
-			products: flatten( pckg.items.map( ( item ) => fill( new Array( item.quantity ), item.product_id ) ) ),
-		} ) ),
-	};
+	let form = getState().shippingLabel.form;
+	const addressesValidation = [];
```

Comment:
Could we name this as `addressValidationQueue` or something that indicates it is a collection of promises?